doc/esmvalcore/preprocessor.rst

```diff
@@ -770,6 +771,26 @@ matches the named regions against the requested string.
 See also :func:`esmvalcore.preprocessor.extract_named_regions`.
 
 
+``extract_shape``
+-------------------------
+
+This function extracts a shape or a representative point for this shape from
+the data. The data outside the selected point(s) will be masked. The data can
+also be clipped to a rectangular region around the shape. This function takes
+the following arguments:
+
+* ``shapefile``: the shapefile containing the geometry of the region to be
+  extracted
+* ``method``: the method to select the region, selecting either all points
+  contained by the shape or one representative point
+* ``clip``: whether to mask the original data set or to clip to a minimal
+  rectangular region containing the shape
+
```

Comment:
this is confusing: clip means restricting the data to a rough rectangle around the desired shape to be selected; this is always useful since data is shrunk to a smaller, yet sufficiently large shape to be able to more efficiently extract the desired shape. Maybe we should phrase it this way, and also, see my previous comment, when will we *not* clip?